CSharp/DevVmPowershell/Helpers/YmlFileHelper.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Dynamic;
+using System.IO;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Text;
+using System.Threading.Tasks;
+using YamlDotNet.RepresentationModel;
+using YamlDotNet.Serialization;
+using YamlDotNet.Serialization.NamingConventions;
+
+namespace Helpers
+{
+	public class YmlFileHelper : IYmlFileHelper
+	{
+		public void UpdateElasticSearchYml()
+		{
+			try
+			{
+				string pathToYmlFile = @"C:\RelativityDataGrid\elasticsearch-main\config\elasticsearch.yml";
```

Comment:
make a copy of the orignal yml file in the same folder with the name elasticsearch_original.yml